submodules/users/users.js

```diff
@@ -4849,12 +4887,20 @@ define(function(require) {
 				user = args.user,
 				needVMUpdate = args.needVMUpdate || true,
 				callback = args.callback,
-				oldPresenceId = args.oldPresenceId || undefined,
-				userExtension = args.userExtension;
+				oldPresenceId = args.oldPresenceId || undefined;
 
 			self.usersListVMBoxesUser(user.id, function(vmboxes) {
-				if (vmboxes.length > 0) {
-					if (needVMUpdate) {
+				if (vmboxes.length === 0) {
+					callback && callback(null);
```

Comment:
You added a check against `_.isEmpty()` once this callback is called, so it is safer to send back empty object here.
```suggestion
					callback && callback({});
```